src/layout/footer/Links.tsx

```diff
@@ -0,0 +1,68 @@
+import Link from 'next/link';
+
+export default function Links() {
+  return (
+    <div className="flex h-58 w-88 flex-col justify-between sm:w-121 md:w-171 md:flex-row">
+      <div className="flex h-29 w-full flex-row justify-between md:w-88">
+        <div className="flex h-29 flex-col justify-between md:h-39">
+          <h3 className="font-bold">Restore Furniture</h3>
+          <ul className="flex h-22 flex-col justify-between md:h-31">
+            <li>
```

Comment:
WDYT of adding an array of objects for mapping this links? Something like 

`const links = [{links: 'https...', displayName: 'Entries'}]`